crane/data.py

```diff
@@ -112,7 +112,7 @@ def load_all(app):
 
     try:
         data_dir = app.config[config.KEY_DATA_DIR]
-        logging.info('loading metadata from %s' % data_dir)
+        logger.info('loading metadata from %s' % data_dir)
```

Comment:
@jortel thanks for the tip! I had not considered that before, and had wondered why the logging API works that way. Makes sense! I will decline from making that change in this particular PR, however, to keep the diff focused only on calling the correct method. I'll make sure to do logging that way in the future though.